lib/structures/Message.js

```diff
@@ -294,6 +295,15 @@ class Message extends Base {
         if(data.stickers !== undefined) {
```

Comment:
Could you add a newline before this if to keep the newlines in Message more consistent?

Reply:
Done, thanks!